doc/user/using-graalvm.md

```diff
@@ -11,12 +11,12 @@ http://www.oracle.com/technetwork/oracle-labs/program-languages/
 
 Inside the GraalVM is a `jre/languages/ruby` directory which has the usual
 structure of a Ruby implementation. It is recommended to add this directory to
-a Ruby manager, see [Configuring Ruby managers](ruby-managers.md) for more
+a Ruby manager, see [configuring Ruby managers](ruby-managers.md) for more
 information.
 
-By default, GraalVM runs TruffleRuby on a JVM, but it can also run it on top
-of the [SubstrateVM](svm.md) for faster startup-time but slower peak-performance.
-This is enabled with the `--native` option.
+By default, GraalVM runs TruffleRuby in the native configuration, which is best
+for short-running processes, but it can also run it on top of the JVM which is
+best for long-running processes. Use `--jvm` to use the JVM configuration.
```

Comment:
I'm don't think using native for long-running processes is problematic. I would use the Java interop + peak performance differentiators here.

Reply:
Fixing.